demo/index.html

```diff
@@ -27,58 +28,35 @@
     <div class="vertical-section-container">
       <h1>auro-popover</h1>
 
-      <p>Component description ...</p>
-
-      <p class="ironIcons">
-        <span class="util_floatLeft util_marginRight--med"
-          >Narrow
-          <iron-icon
-            class="util_marginAuto util_displayBlock"
-            icon="hardware:phone-iphone"
-          ></iron-icon>
-        </span>
-        <span class="util_is-narrowAppears util_floatLeft util_marginRight--med"
-          >Medium
-          <iron-icon
-            class="util_marginAuto util_displayBlock"
-            icon="hardware:tablet-android"
-          ></iron-icon>
-        </span>
-        <span class="util_is-tabletAppears util_floatLeft util_marginRight--med"
-          >Wide
-          <iron-icon
-            class="util_marginAuto util_displayBlock"
-            icon="hardware:tablet"
-          ></iron-icon>
-        </span>
-        <span class="util_is-desktopOnly util_floatLeft util_marginRight--med"
-          >Max
-          <iron-icon
-            class="util_marginAuto util_displayBlock"
-            icon="hardware:tv"
-          ></iron-icon>
-        </span>
-      </p>
-
       <demo-snippet>
         <template>
-          <auro-popover cssClass="testClass">Hello World!</auro-popover>
+            <button id="button1" aria-describedby="tooltip" >Popover bottom</button>
+            <auro-popover for="button1" placement="bottom">Hello World!</auro-popover>
         </template>
       </demo-snippet>
 
-      <h2>Element &#60;auro-popover&#62;</h2>
-
-      <pre><code>class AuroPopover extends LitElement</code></pre>
-
-      <h3>Styling:</h3>
-      <p>Info ...</p>
+      <demo-snippet>
+        <template>
+            <button id="button2" aria-describedby="tooltip">Popover top</button>
```

Comment:
You have the `aria-describedby` but there is no corresponding `ID`? 

Also ... for popovers, is `aria-describedby` needed for a11y? If so, we need to have these in the demo docs.

Reply:
Artifact from a different repository. Removed.